modules/videoio/doc/videoio_overview.markdown

```diff
@@ -32,27 +32,26 @@ Select the backend at runtime
 
 OpenCV automatically selects and uses first available backend (`apiPreference=cv::CAP_ANY`).
 
-As advanced usage you can select the backend to use at runtime. Currently this option is
-available only with %VideoCapture.
```

Comment:
VideoWriter supports this too: https://github.com/opencv/opencv/blame/3.3.0/modules/videoio/include/opencv2/videoio.hpp#L858